src/components/SimplePopup/index.js

```diff
@@ -110,6 +110,7 @@ class SimplePopup extends React.Component<PropsShape> {
               {options.map((option: OptionShape, k: number) => (
                 <div key={k} className={`${baseClass}__footer__item`}>
                   <button
+                    type="button"
```

Comment:
should this not be `options.buttonType` or something like that?

Reply:
there is no form involved, so type should be button